hadoop/README.md

```diff
@@ -1,3 +1,85 @@
+- __WebHCat__:  WebHCat provides a service that a user canuse to run Hadoop MapReduce (or YARN), Pig, Hive jobs. Using this, one can also perform Hive metadata operations using an HTTP (REST style) interface.
+- __Beeline CLI__:  Beeline is one of the ways to connect to Hive through CLI. It is installed alongwith Hive. To establish a Hive connection, enter the following command on the command prompt - beeline -u "jdbc:hive2://jarvis-bootcamp-m:10000" . We can start using Hive, once the connection is established. 
+- __Zeppelin Notebook__:  Zeppelin Notebooks provide an easy way to execute code in a wb notebook. Zeppelin can be used to execute scala, SQL, ssh commands and even schedule a job with cron to run at regular intervals. 
+Using Beeline CLI, it is difficult to manage, execute and present large number of queries. This is one of the advantages of using Zeppelin notebooks. It helps us manage and present queries in a systematic way as the queries are organized into pages and paragraphs.  
+- __Zeppelin Configs__:  Zeppeling runs on master node to establish Hive connection. A customized Hive interpreter is configured based o default JDBC interpreter. 
+- __Zeppelin Usage__:   Zeppelin notebook, in our case, is organized into different paragraphs, each paragraph performing different HiveQL queries. There are few where ssh commands are run after putting %sh at the top of the paragraph. Each paragraph on the page is executed individually and sequentially.
+
+
+
+## Hardware Specifications	
+Hadoop cluster containing 1 master node and 2 worker nodes is created. Each node has 2 cores of 13 GB each and disk space of 100 GB is assigned.
+
+
+## Hive Project
+We have used WORLD DEVELOPMENT INDEX dataset which is dated from 1960 TO 2016. It is available on Google BigQuery and occupies 2.21GB storage space with 21,759,408 rows. 
+The Hive project introduces us how to the execution of Hive queries and Zeppelin notebook and its features. 
+
+### Introduction to Hive Project
+We can divide Hive project in various parts as follows:
+1. Loading data into HDFS
+2. Perform Query optimizations.
+3. Copy data between tables.  
+4. Using Serializer/Deserialize (SerDe) to read data from a tabke and write it back to the HDFS in any custom format. SerDe interface allows user to instruct Hive on how a record should be processed.
+5. Partitioning tables into related parts based on the values of columns.
+6. Performance comparison between SparkQL and HiveQL.
+	 
+### Zeppelin Notebook
+Below is the screenshot of Zeppelin Notebook explaining tasks likes Reading data from Google BigQuery, copying data between tables, loading data into HDFS, creating partitions and comparing for efficiency and various other performance comparisions.
+
+![Zeppelin Notebook Screenshot](./Assets/Zeppelin_Hive.png)
```

Comment:
I don't see where you make the first Hive table, wdi_gs.
Some of your markdown paragraphs aren't rendered.
You can safely skip the select * on the debug table.
You can skip Spark SQL show tables